docs/articles/documentation/guides/continuous-integration/circleci-and-lambdatest.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: docs
+title: Run Tests on LambdaTest with CircleCI
+permalink: /documentation/guides/continuous-integration/circleci-and-lambdatest.html
+---
+# Run Tests on LambdaTest with CircleCI
+
+This topic describes how to integrate TestCafe into the [CircleCI](https://circleci.com/) build process and run tests in the [LambdaTest](https://www.lambdatest.com) cloud testing service.
+
+## Prerequisites
+
+In this tutorial, we will fetch tests from a GitHub repository: [testcafe-ci-demo](https://github.com/DevExpress-Examples/testcafe-ci-demo). Fork this repository before we start.
```

Comment:
do we need to fork or we can just clone?

Reply:
When a user logs in to CircleCI (or Travis), it shows repos that belong to the user or their organization, so I guess they must fork, right?

Reply:
```suggestion
In this tutorial, we fetch tests from a GitHub repository: [testcafe-ci-demo](https://github.com/DevExpress-Examples/testcafe-ci-demo). Fork this repository before you start.
```